bpf/aya-bpf/src/lib.rs

```diff
@@ -57,11 +57,12 @@ pub trait BpfContext {
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn memset(s: *mut u8, c: c_int, n: usize) {
+pub unsafe extern "C" fn memset(s: *mut u8, c: c_int, n: usize) -> u8 {
```

Comment:
This is not needed now that we're making these functions static I think?

We should still fix our memory builtins to 1) be faster 2) work as globals, but we should do that in a separate PR and for all of them

Reply:
You are right ! It works without. I will change that

Reply:
Done